src/Symfony/Component/Mailer/Bridge/Mailjet/Transport/MailjetApiTransport.php

```diff
@@ -106,6 +106,12 @@ private function getPayload(Email $email, Envelope $envelope): array
         if ($emails = $email->getBcc()) {
             $message['Bcc'] = $this->formatAddresses($emails);
         }
+        if ($emails = $email->getReplyTo()) {
+            if (1 < $length = \count($emails)) {
+                $this->getLogger()->warning(sprintf('Mailjet\'s API only supports one Reply-To email, %d given. The first one will be used.', $length));
```

Comment:
I would throw an exception instead. I think that having more than one `Reply-To` address is an edge case and if it happens, it's important to make sure that the dev understands that it is not supported. I understand the interoperability argument, but I think it's better here to fail when something is not supported.

Can you also make the same change in Sendgrid to be consistent?

Reply:
@fabpot wouldn't adding this change to Sengrid break backward-compatibility as some apps may rely on the current behaviour of the transport ?

Also, if transports are not fully interoperable, should we document it and maybe list transport specificities ?